bgpd/bgp_pbr.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * BGP pbr
+ * Copyright (C) 6WIND
+ *
+ * This file is part of FRR.
+ *
+ * FRR is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2, or (at your option) any
+ * later version.
+ *
+ * FRR is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; see the file COPYING; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+#ifndef __BGP_PBR_H__
+#define __BGP_PBR_H__
+
+#include "nexthop.h"
+#include "zclient.h"
+
+struct bgp_pbr_match {
+	char ipset_name[ZEBRA_IPSET_NAME_SIZE];
+
+	/* mapped on enum ipset_type
```

Comment:
Let's use a consistent comment style throughout the file.

Reply:
- removed tabs between #define and the variable name

Reply:
The comments throughout this file need to be fixed to the /* by itself on a line -- other than this, the logic looks good to me. Please fix the comments so we can push this. :-)

Reply:
Yep, again, let's use a consistent comment style throughout the file.